### PR TITLE
docs: improve PR workflow

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add "If you sent to the wrong branch"
- add optional step to back up the PR branch

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

**Review by datamweb — APPROVED** [review]
💯

File: contributing/workflow.md [hunk 1]
```diff
@@ -276,6 +282,54 @@ And finally push your local branch to your GitHub repository:
 > git push --force-with-lease origin fix/problem123
 ```
 
+## If you sent to the wrong branch
+
+If you have sent a PR to the wrong branch, you need to create a new PR branch.
+
+When you have the PR branch `feat-abc` and you should have sent the PR to `4.3`,
+but you created the PR branch from `develop` and sent a PR.
+
```
Comment (MGatner) [inline contributing/workflow.md:291]: What about helping them find the commit IDs before they get to the point of needing them?
```suggestion
Copy the IDs of any commits you made that you want to keep:

`git log --no-decorate`

```
I cannot add triple backticks to the suggestion, but you get the idea.

**Review by MGatner — APPROVED** [review]
Thanks for adding this! Seems a frequent need. One suggestion, take it or leave it.

File: contributing/workflow.md (hunk 1 above)
Comment (kenjis) [reply contributing/workflow.md:291]: Added `git log`.

`--no-decorate ` does not Print out the ref names of any commits that are shown.
https://git-scm.com/docs/git-log#_options
It makes difficult to know the branch HEAD.

**Review by paulbalandan — APPROVED** [review]

File: contributing/workflow.md (hunk 1 above)
Comment (MGatner) [reply contributing/workflow.md:291]: Oops! Thanks for looking that up. I was on mobile and trying to find the log command with the least unnecessary output, should've just stuck with the vanilla!

**Pull request merged**